templates/customgroup.tpl

```diff
@@ -0,0 +1,18 @@
+{* FILE: customfieldnames/templates/customgroupfield.tpl to add custom field for custom data set*}
+<table id="custom-fields" class="form-layout">
```

Comment:
for the id, let's use "lcd-custom-fields" to ensure no conflicts with core. change in the js below as well

Reply:
Done